sdk/storage/azure-storage-file-share/swagger/src/main/java/ShareStorageCustomization.java

```diff
@@ -32,7 +32,9 @@ public void customize(LibraryCustomization customization, Logger logger) {
         changeJacksonXmlRootElementName(models.getClass("SourceModifiedAccessConditions"), "source-modified-access-conditions");
 
         ClassCustomization shareTokenIntent = models.getClass("ShareTokenIntent");
-        shareTokenIntent.getJavadoc().setDescription("The request intent for whether the file should be backed up.");
+        shareTokenIntent.getJavadoc().setDescription("The request intent for whether the file should be backed up. " +
+            "Possible values are: {@link ShareTokenIntent#BACKUP} - specifies requests that are intended for backup/admin type operations,\n" +
+            " meaning that all file/directory ACLs are bypassed and full permissions are granted. User must also have required RBAC permission.");
```

Comment:
Why are we overriding the javadoc in the first place?

Reply:
I added more description since before it was just "Defines values for ShareTokenIntent." I thought this wasn't descriptive enough for the user to understand what ShareTokenIntent is for. Do you think it would be better to remove the description from the ShareTokenIntent class and have the description describing the behavior in the client builders?